lang/lang_en_de.txt

```diff
@@ -1241,4 +1241,196 @@
 
 #MSG_OFF
 " [off]"
+" [aus]"
+
+#
+"Back"
+"Zurueck"
+
+#
+"Checks"
+"\x00"
+
+#
+"False triggering"
+"Falschtriggerung"
+
+#
+"FINDA:"
+"\x00"
+
+#
+"Firmware   [none]"
+"Firmware   [ohne]"
+
+#
+"Firmware [strict]"
+"Firmware [streng]"
+
+#
+"Firmware   [warn]"
+"\x00"
+
+#
+"HW Setup"
+"HW Einstellungen"
+
+#
+"IR:"
+"\x00"
+
+#
+"Magnets comp.[N/A]"
+"Magnet Komp.  [nv]"
+
+#
+"Magnets comp.[Off]"
+"Magnet Komp. [Aus]"
+
+#
+"Magnets comp. [On]"
+"Magnet Komp.  [An]"
+
+#
+"Mesh         [3x3]"
+"\x00"
+
+#
+"Mesh         [7x7]"
+"\x00"
+
+#
+"Mesh bed leveling"
+"Mesh Bett Ausgleich"
+
+#
+"MK3S firmware detected on MK3 printer"
+"MK3S-Firmware auf MK3-Drucker erkannt"
+
+#
+"MMU Mode [Normal]"
+"MMU Modus[Normal]"
+
+#
+"MMU Mode[Stealth]"
+"MMU Mod.[Stealth]"
+
+#
+"Mode change in progress ..."
+"Moduswechsel erfolgt..."
+
+#
+"Model      [none]"
+"Modell     [ohne]"
+
+#
+"Model    [strict]"
+"Modell   [streng]"
+
+#
+"Model      [warn]"
+"Modell     [warn]"
+
+#
+"Nozzle d.  [0.25]"
+"Duese D.   [0.25]"
+
+#
+"Nozzle d.  [0.40]"
+"Duese D.   [0.40]"
+
+#
+"Nozzle d.  [0.60]"
+"Duese D.   [0.60]"
+
+#
+"Nozzle     [none]"
+"Duese      [ohne]"
+
+#
+"Nozzle   [strict]"
+"Duese    [streng]"
+
+#
+"Nozzle     [warn]"
+"Duese      [warn]"
+
+#
+"G-code sliced for a different level. Continue?"
+"G-Code ist fuer einen anderen Level geslict. Fortfahren?"
+
+#
+"G-code sliced for a different level. Please re-slice the model again. Print cancelled."
+"G-Code ist fuer einen anderen Level geslict. Bitte slicen Sie das Modell erneut. Druck abgebrochen."
+
+#
+"G-code sliced for a different printer type. Continue?"
+"G-Code ist fuer einen anderen Drucker geslict. Fortfahren?"
+
+#
+"G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
+"G-Code ist fuer einen anderen Drucker geslict. Bitte slicen Sie das Modell erneut. Druck abgebrochen."
+
+#
+"G-code sliced for a newer firmware. Continue?"
+"G-Code ist fuer eine neuere Firmware geslict. Fortfahren?"
+
+#
+"G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
+"G-Code ist fuer eine neuere Firmware geslict. Bitte die Firmware updaten. Druck abgebrochen."
+
+#
+"PINDA:"
+"\x00"
+
+#
+"Preheating to cut"
+"Heizen zum Schnitt"
+
+#
+"Preheating to eject"
+"Heizen zum Auswurf"
+
+#
+"Printer nozzle diameter differs from the G-code. Continue?"
+"Der Durchmesser der Druckerduese weicht vom G-Code ab. Fortfahren?"
+
+#
+"Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
+"Der Durchmesser der Druckerduese weicht vom G-Code ab. Bitte ueberpruefen Sie den Wert in den Einstellungen. Druck abgebrochen."
+
+#
+"Rename"
+"Umbenennen"
+
+#
+"Select"
+"Auswahl"
+
+#
+"Sensor info"
+"Sensor Info"
+
+#
+"Sheet"
+"Blech"
+
+#
+"Sound     [assist]"
```

Comment:
After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]